### PR TITLE
DiseasesExpanded - New strings for translation and event critters fix

### PR DESCRIPTION
**Pull request by realwiseserpent**
Hi! I moved strings from unspecified flask and twitch event objects and critters to STRINGS class so they could be translated. 
Also small fix for event critter

### CODE REVIEW

File: DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public GameObject CreatePrefab()
...
+            EntityTemplates.AddCreatureBrain(placedEntity, new ChoreTable.Builder().Add((StateMachine.BaseDef)new DeathStates.Def()).Add((StateMachine.BaseDef)new TrappedStates.Def()).Add((StateMachine.BaseDef)new BaggedStates.Def()).Add((StateMachine.BaseDef)new FallStates.Def()).Add((StateMachine.BaseDef)new StunnedStates.Def()).Add((StateMachine.BaseDef)new DrowningStates.Def()).Add((StateMachine.BaseDef)new DebugGoToStates.Def()).Add((StateMachine.BaseDef)new FleeStates.Def()).Add((StateMachine.BaseDef)new DropElementStates.Def()).Add((StateMachine.BaseDef)new IdleStates.Def()), GameTags.Creatures.Species.PuftSpecies, (string)null);
```
Comment (pether-pg) [inline DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs:45]: They were glomSpecies by design to have all of them in one category

File: DiseasesExpanded/RandomEvents/Configs/HarmlessBeeConfig.cs [hunk 2]
```diff
@@ -36,7 +35,7 @@ public GameObject CreatePrefab()
...
+            EntityTemplates.AddCreatureBrain(placedEntity, new ChoreTable.Builder().Add((StateMachine.BaseDef)new DeathStates.Def()).Add((StateMachine.BaseDef)new TrappedStates.Def()).Add((StateMachine.BaseDef)new BaggedStates.Def()).Add((StateMachine.BaseDef)new FallStates.Def()).Add((StateMachine.BaseDef)new StunnedStates.Def()).Add((StateMachine.BaseDef)new DrowningStates.Def()).Add((StateMachine.BaseDef)new DebugGoToStates.Def()).Add((StateMachine.BaseDef)new FleeStates.Def()).Add((StateMachine.BaseDef)new DropElementStates.Def()).Add((StateMachine.BaseDef)new IdleStates.Def()), GameTags.Creatures.Species.BeetaSpecies, (string)null);
```
Comment (pether-pg) [inline DiseasesExpanded/RandomEvents/Configs/HarmlessBeeConfig.cs:38]: please revert

**Review by pether-pg — CHANGES_REQUESTED** [review]

File: DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs (hunk 1 above)
Comment (realwiseserpent) [reply DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs:45]: ok, mb they should have their own category?

Comment (pether-pg) [reply DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs:45]: gloms are good enough and currently having 1 non-modded critter, so it's good. they also work kinda similar

Comment (realwiseserpent) [reply DiseasesExpanded/RandomEvents/Configs/FluffyPuffyConfig.cs]: Fixed.

File: DiseasesExpanded/RandomEvents/Configs/HarmlessBeeConfig.cs (hunk 2 above)
Comment (realwiseserpent) [reply DiseasesExpanded/RandomEvents/Configs/HarmlessBeeConfig.cs]: Fixed.

**Review by pether-pg — APPROVED** [review]

**Pull request merged**